PowerEditor/src/WinControls/Preference/preference.rc

```diff
@@ -84,15 +84,27 @@ BEGIN
     CONTROL         "Default",IDC_RADIO_LWDEF,"Button",BS_AUTORADIOBUTTON | WS_GROUP,275,21,59,10
     CONTROL         "Aligned",IDC_RADIO_LWALIGN,"Button",BS_AUTORADIOBUTTON,275,36,60,10
     CONTROL         "Indent",IDC_RADIO_LWINDENT,"Button",BS_AUTORADIOBUTTON,275,51,62,10
-    CONTROL         "Enable Multi-Editing (Ctrl+Mouse click/selection)",IDC_CHECK_MULTISELECTION,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,90,270,10
-    CONTROL         "Enable current line highlighting",IDC_CHECK_CURRENTLINEHILITE,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,103,270,10
-    CONTROL         "Enable smooth font",IDC_CHECK_SMOOTHFONT,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,116,250,10
-    CONTROL         "Enable virtual space",IDC_CHECK_VIRTUALSPACE,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,129,270,10
-    CONTROL         "Enable scrolling beyond last line",IDC_CHECK_SCROLLBEYONDLASTLINE,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,142,270,10
-    CONTROL         "Keep selection when right-click outside of selection",IDC_CHECK_RIGHTCLICKKEEPSSELECTION,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,155,270,10
-    CONTROL         "Disable advanced scrolling feature due to touchpad issue",IDC_CHECK_DISABLEADVANCEDSCROLL,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,168,270,10
+    CONTROL         "Enable Multi-Editing (Ctrl+Mouse click/selection)",IDC_CHECK_MULTISELECTION,
+                    "Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,90,270,10
+    CONTROL         "Enable current line highlighting",IDC_CHECK_CURRENTLINEHILITE,
+                    "Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,103,270,10
+    CONTROL         "Enable current line frame",IDC_CARETLINEFRAME_WIDTH_CHECK,
+                    "Button",BS_AUTOCHECKBOX | WS_TABSTOP,68,116,180,10
+    LTEXT           "Width :",IDC_CARETLINEFRAME_WIDTH_STATIC,256,116,37,8,0,WS_EX_RIGHT
+    CONTROL         "",IDC_CARETLINEFRAME_WIDTH_SLIDER,"msctls_trackbar32",TBS_AUTOTICKS | TBS_BOTH | TBS_NOTICKS | TBS_TRANSPARENTBKGND | WS_TABSTOP,294,116,57,13
+    LTEXT           "1",IDC_CARETLINEFRAME_WIDTH_DISPLAY,353,116,12,8
+    CONTROL         "Enable smooth font",IDC_CHECK_SMOOTHFONT,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,129,250,10
+    CONTROL         "Enable virtual space",IDC_CHECK_VIRTUALSPACE,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,142,270,10
+    CONTROL         "Enable scrolling beyond last line",IDC_CHECK_SCROLLBEYONDLASTLINE,
+                    "Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,155,270,10
+    CONTROL         "Keep selection when right-click outside of selection",IDC_CHECK_RIGHTCLICKKEEPSSELECTION,
+                    "Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,168,270,10
+    CONTROL         "Disable advanced scrolling feature due to touchpad issue",IDC_CHECK_DISABLEADVANCEDSCROLL,
+                    "Button",BS_AUTOCHECKBOX | WS_TABSTOP,54,181,270,10
 END

```

Comment:
Please copy/paste the following codes:
```
IDD_PREFERENCE_SUB_EDITING DIALOGEX 0, 0, 455, 203
STYLE DS_SETFONT | DS_FIXEDSYS | DS_CONTROL | WS_CHILD
FONT 8, "MS Shell Dlg", 0, 0, 0x1
BEGIN
    GROUPBOX        "Caret Settings",IDC_CARETSETTING_STATIC,25,6,156,67,BS_CENTER
    LTEXT           "Width :",IDC_WIDTH_STATIC,53,27,37,8,0,WS_EX_RIGHT
    COMBOBOX        IDC_WIDTH_COMBO,97,25,40,30,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
    RTEXT           "Blink rate :",IDC_BLINKRATE_STATIC,28,52,60,8
    CONTROL         "",IDC_CARETBLINKRATE_SLIDER,"msctls_trackbar32",TBS_AUTOTICKS | TBS_BOTH | TBS_NOTICKS | TBS_TRANSPARENTBKGND | WS_TABSTOP,105,52,57,13
    LTEXT           "S",IDC_CARETBLINKRATE_S_STATIC,166,52,12,8
    LTEXT           "F",IDC_CARETBLINKRATE_F_STATIC,93,52,12,8,0,WS_EX_RIGHT

    GROUPBOX        "Line Wrap",IDC_LW_GB_STATIC,193,6,78,67,BS_CENTER
    CONTROL         "Default",IDC_RADIO_LWDEF,"Button",BS_AUTORADIOBUTTON | WS_GROUP,203,21,59,10
    CONTROL         "Aligned",IDC_RADIO_LWALIGN,"Button",BS_AUTORADIOBUTTON,203,36,60,10
    CONTROL         "Indent",IDC_RADIO_LWINDENT,"Button",BS_AUTORADIOBUTTON,203,51,62,10

    GROUPBOX        "Current Line Mark",IDC_CURRENTLINEMARK_STATIC,284,6,129,83,BS_CENTER
    CONTROL         "None",IDC_RADIO_CLM_NONE,"Button",BS_AUTORADIOBUTTON | WS_GROUP,307,22,59,10
    CONTROL         "Highlight",IDC_RADIO_CLM_HILITE,"Button",BS_AUTORADIOBUTTON,307,37,60,10
    CONTROL         "Frame",IDC_RADIO_CLM_FRAME,"Button",BS_AUTORADIOBUTTON,307,52,62,10
    LTEXT           "Width :",IDC_CARETLINEFRAME_WIDTH_STATIC,299,67,37,8,0,WS_EX_RIGHT
    CONTROL         "",IDC_CARETLINEFRAME_WIDTH_SLIDER,"msctls_trackbar32",TBS_AUTOTICKS | TBS_BOTH | TBS_NOTICKS | TBS_TRANSPARENTBKGND | WS_TABSTOP,337,67,57,13
    LTEXT           "1",IDC_CARETLINEFRAME_WIDTH_DISPLAY,396,67,12,8
	
    CONTROL         "Enable Multi-Editing (Ctrl+Mouse click/selection)",IDC_CHECK_MULTISELECTION,
                    "Button",BS_AUTOCHECKBOX | WS_TABSTOP,26,114,270,10
    CONTROL         "Enable smooth font",IDC_CHECK_SMOOTHFONT,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,26,129,250,10
    CONTROL         "Enable virtual space",IDC_CHECK_VIRTUALSPACE,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,26,142,270,10
    CONTROL         "Enable scrolling beyond last line",IDC_CHECK_SCROLLBEYONDLASTLINE,
                    "Button",BS_AUTOCHECKBOX | WS_TABSTOP,26,155,270,10
    CONTROL         "Keep selection when right-click outside of selection",IDC_CHECK_RIGHTCLICKKEEPSSELECTION,
                    "Button",BS_AUTOCHECKBOX | WS_TABSTOP,26,168,270,10
    CONTROL         "Disable advanced scrolling feature due to touchpad issue",IDC_CHECK_DISABLEADVANCEDSCROLL,
                    "Button",BS_AUTOCHECKBOX | WS_TABSTOP,26,181,270,10
END
```

to have the new look:

![image](https://user-images.githubusercontent.com/90293/163235131-d835f570-78cc-4416-a809-5d33536d00c1.png)

Reply:
_Mark_ in _Current Line Mark_ might be poor wording, as the UI already has a lot of uses for this word.

How about _Current Line Emphasis_ for the title of this groupbox?

Also, _Highlight_ on its own is now weak -- what about _Colour Background_ instead?

In _Style Configurator_ the wording is _Current line background colour_.  That's fine but user needs to be aware that this will also serve for the _Frame_ colour.  I'm not saying there needs to be any change to _Style Configurator_, I'm just pointing this out.  Perhaps it is best handled in user manual.

Reply:
@alankilborn,

> Mark in Current Line Mark might be poor wording, as the UI already has a lot of uses for this word.

I agree.

> How about Current Line Emphasis for the title of this groupbox?

Translating "Emphasis" to my native language, it's not the best term.
But, if you're a native English speaker, you know better. :)

> In Style Configurator the wording is Current line background colour.

That's a good point.

So the title can be "Current line background colour".
And then:
"None"
"Entire line"
"Frame only"

If "Current line background colour" is too long, "Current line highlighting" seems good too.